libimage/manifests/manifests.go

```diff
@@ -547,6 +556,8 @@ func (l *list) Add(ctx context.Context, sys *types.SystemContext, ref types.Imag
 					OSFeatures:     append([]string{}, platform.OSFeatures...),
 					Size:           instance.Size,
 					ArtifactType:   instance.ArtifactType,
+					Annotations:    mapToSlice(instance.Annotations),
+					URLs:           slices.Clone(instance.URLs),
```

Comment:
is there a reason for the clone? I cannot see anything modifying the slice so I don't think clone is needed.

Reply:
Changed it.